bounties/npm/s3-kilatstorage/1/vulnerability.json

```diff
@@ -0,0 +1,55 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2021-01-26",
+    "AffectedVersionRange": "*",
+    "Summary": "Command Injection",
+    "Contributor": {
+        "Discloser": "tanimuzair",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "KilatStorages3",
```

Comment:
This needs to match the exact name of the `npm` package.

Reply:
Thank you. That worked.